roles/pulp/defaults/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+pulp_image: quay.io/pulp/pulp:stable
+pulp_ports:
+  - "8080:80"
+pulp_volumes:
+  - /var/lib/pulp/settings:/etc/pulp:Z
+  - /var/lib/pulp/pulp_storage:/var/lib/pulp:Z
+  - /var/lib/pulp/pgsql:/var/lib/pgsql:Z
+  - /var/lib/pulp/containers:/var/lib/containers:Z
```

Comment:
`/var/lib/containers` is a path that is owned and used by already by container related tooling, I don't think we should mess with that and instead keep this scoped to `/var/lib/pulp/containers`

Reply:
That's a path *inside* the Pulp container, and that's following their docs:
https://pulpproject.org/pulp-oci-images/docs/admin/reference/available-images/multi-process-images/